docs/source/design/testing.rst

```diff
@@ -0,0 +1,99 @@
+.. -*- mode: rst -*-
+.. vi: set ft=rst sts=4 ts=4 sw=4 et tw=79:
+
+.. _chap_design_testing:
+
+**********************************
+Continuous integration and testing
+**********************************
+
+.. topic:: Specification scope and status
+
+   This specification describes the current implementation.
+
+DataLad is tested using a pytest-based testsuite that is run locally and via continuous integrations setups.
+Code development should ensure that old and new functionality is appropriately tested.
+The project aims for good unittest coverage (at least 80%).
+
+Running tests
+=============
+
+``datalad/tests`` contains tests for the core portion of the project.
+More tests are provided under corresponding submodules in ``tests/`` subdirectories to simplify re-running the tests concerning that portion of the codebase.
+The test suite is run using
+
+.. code-block:: bash
+
+   pytest datalad
+   # or, with coverage reports
+   pip install pytest coverage
+   pytest --cov=datalad datalad
```

Comment:
taking into account above and that for this you need `pytest-cov` AFAIK, not just coverage, and that we need to point to tox.ini, a more complete IMHO recipe would be

```suggestion
   pip install -e .[tests]
   python -m pytest -c tox.ini datalad
   # or, with coverage reports
   python -m pytest  -c tox.ini --cov=datalad datalad
```

if we agree to use `python -m pytest` instead of `pytest` -- then snippets below shoul also be adjusted. If not -- then this snippet should be 

```suggestion
   pip install -e .[tests]
   pytest -c tox.ini datalad
   # or, with coverage reports
   pytest  -c tox.ini --cov=datalad datalad
```

Reply:
Thx, I feel it makes sense to use ``python -m pytest...`` given your explanation!